lib/everypolitician.rb

```diff
@@ -74,7 +74,7 @@ def legislature(query)
       legislature = legislatures.find do |l|
         query.all? { |k, v| l.__send__(k).to_s.downcase == v.to_s.downcase }
       end
-      fail Error, "Unknown legislature: #{query}" if legislature.nil?
+      return if legislature.nil?
       legislature
```

Comment:
Ha, yep, good point! Fixed in https://github.com/everypolitician/everypolitician-ruby/pull/40/commits/da3906b07c0c82a8e0c72fa1bca31965d8d0a7c5.